src/test/java/com/artipie/rpm/RpmOptionsTest.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for {@link RpmOptions}.
+ *
+ * @since 0.10
+ */
+@SuppressWarnings("PMD.TooManyMethods")
+public class RpmOptionsTest {
+
+    @Test
+    void createsCorrectDigestArgName() {
```

Comment:
@paulodamaso I think it would be way better to create a custom matcher that will assert options to avoid code duplications and multiple test.
Than we can have something like:
```java
MatcherAssert.assertThat(
     RpmOptions.DIGEST.option(),
     new IsOptionEqualTo(expectedOption)
)
```

Reply:
@Vatavuk I like the suggestion, I think we can simply use `AllOf(MatcherOf(option -> ...), MatcherOf(option -> ...), ...)` and do not create custom matcher.